changes/4123.fixed

```diff
@@ -0,0 +1 @@
+Fixed custom fields not auto-populating when creating objects through the ORM.
```

Comment:
Might be good to add a note in `nautobot/docs/models/extras/customfield.md` explicitly calling out the need to use `validated_save` rather than `save` when creating model instances that may have custom fields?

Actually I see there's already a note to this effect in `nautobot/docs/user-guides/custom-fields.md`. Duplication of documentation strikes again. Still might be good to copy-paste it into the other doc as well.

Reply:
What do you think about moving any ORM interaction from the docs into the user guide and then saying "if you want to interact with the ORM, check out the corresponding `[User Guide](...)`"?

Reply:
Sure, cross-referencing instead of duplicating is definitely good. :)

Reply:
Should be addressed, take a look if it works for you.